Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Multiplayer Samples Co-op Changelog
 
+## [unreleased] - aaaa-mm-dd
```

Comment:
a for año?
:P

Reply:
you can resolve this, this isn't a merge blocker :P

Reply:
I took this from the Boss Room changelog 😂 It might have been haha.